README.md

```diff
@@ -1,2 +1,76 @@
 # Countries
 This app displays exciting information and facts from countries all over the world
+
+## Built With
+
+- REACT.JS
+- REDUX
+- TAILWIND-CSS
+
+## Install
+### Run the project in your local machine
+
+- Clone the repository
+```
+  $ git clone https://github.com/Estherstarr/countriesAPI.git
+  $ cd countriesAPI
+```
+
+- Install all dependancies
+```
+$ npm install
+```
+
+- Run the server
+```
+$ npm start
+```
+- Open your browser and go to [localhost:3000](http://localhost:3000/)
+
+
+### Run Linters tests
+To track linter errors locally follow these steps:  
+
+Download all the dependencies run:
+```
+$ npm install
+```
+Track HTML linter errors run:
+```
+$ npx hint .
+```
+Track CSS linter errors run:
+```
+$ npx stylelint "**/*.{css,scss}"
+```
+Track JavaScript linter errors run:
+```
+$ npx eslint .
+```
+
+### Deployment
+[Live Demo Link](https://isnt-jabokpoe-awesome.netlify.app/)
+
+## Author
+
+👤 **Author : ONWUANIBE ONOMEH**
+
+- GitHub: [@githubhandle](https://github.com/Estherstarr/)
+- Twitter: [@twitterhandle](https://twitter.com/AnibeEsther)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/onwuanibeonome)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+ Nelson Sakwa the author of the original design used in this project.
+
+## 📝 License
+
+This project is [https://github.com/git/git-scm.com/blob/main/MIT-LICENSE.txt](./MIT.md) licensed.
```

Comment:
[OPTIONAL] Although you used a working URL on the link name, the link doesn't work because you placed the link in the square brackets `[ ]` instead of inside the parenthesis `()`. Therefore, you should be having something like this:

```
    [MIT-LICENSE](https://github.com/git/git-scm.com/blob/main/MIT-LICENSE.txt)
```